lib/blockscore/connection.rb

```diff
@@ -20,9 +26,9 @@ def delete(path, params)
 
     def headers
       @headers ||= {
-        'Accept' => "application/vnd.blockscore+json;version=#{BlockScore::VERSION.to_i}",
-        'User-Agent' => "blockscore-ruby/#{BlockScore::VERSION} (https://github.com/BlockScore/blockscore-ruby)",
-        'Content-Type' => 'application/json'
+        'Accept' => ACCEPT_HEADER,
+        'User-Agent' => USER_AGENT,
+        'Content-Type' => CONTENT_TYPE
```

Comment:
This is a small preference, but I would generally align the `=>` in a Hash assignment. I find it easier to read, and easier to edit.

The only downside is when you add a longer key, or remove the longest key and have to reformat the Hash, but that doesn't seem to be a big deal in practice.